recipe/build-r-base.sh

```diff
@@ -301,6 +301,8 @@ Mingw_w64_makefiles() {
     export CPATH=${PREFIX}/Library/include
     export LIBRARY_PATH=${PREFIX}/Library/lib
 
+    # Support older BLAS implementations (pre-2025)
+    sed -i '/^LIBSOURCES =/ s/$/ dgemmtr.f zgemmtr.f/' src/modules/lapack/Makefile.win
```

Comment:
That's going to be a lot slower than intended.

Reply:
Is removing the symbols preferable to the included patch which adds the symbols and it's working?

Reply:
Yes, removing them is preferable. can you send a PR to remove them?